azure-iot-device/azure/iot/device/provisioning/aio/async_provisioning_device_client.py

```diff
@@ -17,31 +17,34 @@
 from azure.iot.device.provisioning.abstract_provisioning_device_client import (
     log_on_register_complete,
 )
-from azure.iot.device.provisioning.internal.polling_machine import PollingMachine
+from azure.iot.device.iothub.pipeline import exceptions as pipeline_exceptions
```

Comment:
There should be a `azure.iot.device.provisioning.pipeline.exceptions` file similar to the one in iothub, rather than using the same file. Nothing in provisioning should need anything from iothub